Core/src/org/sleuthkit/autopsy/modules/photoreccarver/PhotoRecCarverFileIngestModule.java

```diff
@@ -222,13 +224,13 @@ public void startUp(IngestJobContext context) throws IngestModule.IngestModuleEx
 
             // Scan the file with Unallocated Carver.
             ProcessBuilder processAndSettings = new ProcessBuilder(
-                    "\"" + executableFile + "\"",
+                    executableFile.toString(),
```

Comment:
What was the logic behind removing quotes around strings? Does this continue to work on Windows when there are spaces in the paths?